js/tc39/tc39_test.go

```diff
@@ -51,6 +51,45 @@ var (
 		"Array.prototype.item",        // not even standard yet
 		"TypedArray.prototype.item",   // not even standard yet
 		"String.prototype.replaceAll", // not supported at all, Stage 4 since 2020
+
+		// from goja
+		"Symbol.asyncIterator",
+		"async-functions",
+		"class-static-block",
+		"class-fields-private",
+		"class-fields-private-in",
+		"regexp-named-groups",
+		"regexp-dotall",
+		"regexp-unicode-property-escapes",
+		"regexp-unicode-property-escapes",
+		"regexp-match-indices",
+		"legacy-regexp",
+		"tail-call-optimization",
+		"Temporal",
+		"import-assertions",
+		"dynamic-import",
+		"logical-assignment-operators",
+		"coalesce-expression",
+		"import.meta",
+		"optional-chaining",
+		"Atomics",
+		"Atomics.waitAsync",
+		"FinalizationRegistry",
+		"WeakRef",
+		"numeric-separator-literal",
+		"Object.fromEntries",
+		"Object.hasOwn",
+		"__getter__",
+		"__setter__",
+		"ShadowRealm",
+		"SharedArrayBuffer",
+		"error-cause",
+		"resizable-arraybuffer", // stage 3 as of 2021 https://github.com/tc39/proposal-resizablearraybuffer
+
```

Comment:
Out of curiosity, is this newline relevant for separating something? If yes, I would add a comment as you did before for `goja`.

Reply:
just to separate them, the above list is from goja, while the below is stuff I added (goja doesn't run those through different means)